packages/olo-gulp-helpers/package.json

```diff
@@ -34,7 +34,7 @@
     "ts-loader": "6.2.1",
     "tslint": "5.9.1",
     "tslint-teamcity-reporter": "2.0.0",
-    "typescript": "2.7",
+    "typescript": "3.9.7",
```

Comment:
(this is a public repo)

This looks fine, but we'll need to check our internal projects to make sure this package bump won't break any builds.

Reply:
Understand. This needed to be bumped to a 3.x version to get the build to pass for `data-generator`, and as such I chose the latest version. Is there some sort of reverse-lookup to see which projects of ours use the `olo-gulp-helpers`?